automation/test.sh

```diff
@@ -32,27 +32,21 @@ export WORKSPACE="${WORKSPACE:-$PWD}"
 readonly ARTIFACTS_PATH="$WORKSPACE/exported-artifacts"
 readonly TEMPLATES_SERVER="https://templates.ovirt.org/kubevirt/"
 
-if [[ $TARGET =~ openshift-.* ]]; then
+if [[ $TARGET == windows2016 ]]; then
+  export KUBEVIRT_PROVIDER="k8s-1.11.0"
+elif [[ $TARGET =~ os-.* ]]; then
   # when testing on slow CI system cleanup sometimes takes very long.
   # openshift clusters are more memory demanding. If the cleanup
   # of old vms does not go fast enough they run out of memory.
   # To still allow continuing with the tests, give more memory in CI.
   export KUBEVIRT_MEMORY_SIZE=6144M
-  if [[ $TARGET =~ .*-crio ]]; then
-    export KUBEVIRT_PROVIDER="os-3.11.0-crio"
-  elif [[ $TARGET =~ .*-multus ]]; then
-    export KUBEVIRT_PROVIDER="os-3.11.0-multus"
-  else
-    export KUBEVIRT_PROVIDER="os-3.11.0"
-  fi
-elif [[ $TARGET =~ .*-1.10.4 ]]; then
-  export KUBEVIRT_PROVIDER="k8s-1.10.11"
-elif [[ $TARGET =~ .*-multus-1.13.3 ]]; then
-  export KUBEVIRT_PROVIDER="k8s-multus-1.13.3"
-elif [[ $TARGET =~ .*-genie-1.11.1 ]]; then
-  export KUBEVIRT_PROVIDER="k8s-genie-1.11.1"
-else
-  export KUBEVIRT_PROVIDER="k8s-1.11.0"
+fi
+
+export KUBEVIRT_PROVIDER=$TARGET
```

Comment:
I think you're overwriting the windows provider here

Reply:
true need to change order